tools/scenario-player/example-scenarios/scenario-example-v2.yaml

```diff
@@ -13,7 +13,7 @@ settings:
   #chain: any
   ## Other examples
   #chain: ropsten
-  #chain: kovan
```

Comment:
why make this example specific to kovan?

Reply:
Ah, thats a mistake. Thanks.